src/app/_components/best-selling-section/best-selling.tsx

```
@@ -0,0 +1,68 @@
+         {/* REUSABLE SUBTITLE */}
+          <Subtitle text="Best Selling" />
+           {/* HEADING */}
+          <h2 className={cn("text-3xl font-bold text-maroon-700")}>
+            <span className={cn("text-softpink-500")}>Check Out</span> What <br />
+            Everyone’s <span className={cn("text-softpink-500")}>Buying</span> Right Now
+          </h2>
+           {/* DESCRIPTION */}
+          <p className={cn("text-zinc-500 text-base")}>
+            Not sure what to choose?<br />
+            Start with our best sellers, these are the gifts our customers keep coming back for.
+            Whether you're celebrating a birthday, anniversary or wedding, our top picks are guaranteed to leave a lasting impression. 
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)